docs/digger/admin/installing-jenkins-plugins.asciidoc

```diff
@@ -1,12 +1,20 @@
 == Installing Jenkins plugins
-Digger will by default install a number of Jenkins plugins required for its proper operation. If more Jenkins plugins are required, they can be installed by including them in Ansible playbook or using the Jenkins UI.
-
 NOTE: Plugins for Jenkins can be found at https://plugins.jenkins.io
 
+Digger will by default install a number of Jenkins plugins required for its proper operation. If more Jenkins plugins are required, they can be installed by including them as a task in Ansible playbook or using the Jenkins UI.
```

Comment:
could add `digger installer will .....for its proper operation in the context of digger`

Reply:
The sentence is currently `Digger will by default install a number of Jenkins plugins required for it's proper operation`. Change this to `Digger Installer will by default by default install a number of Jenkins plugins required for it's proper operation in digger`

Reply:
resolved